server.js

```diff
@@ -2781,6 +2781,61 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CircleCI build integration.
+// https://circleci.com/api/v1/project/BrightFlair/PHP.Gt?circle-token=0a5143728784b263d9f0238b8d595522689b3af2&limit=1&filter=completed
+camp.route(/^\/circleci\/project\/([^\/]+\/[^\/]+)(?:\/(.*))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `doctrine/orm`.
+  var branch = match[2];
+  var format = match[3];
+
+  var apiUrl = 'https://circleci.com/api/v1/project/' + userRepo;
+  if(branch != null) {
+    apiUrl +=
+      "/tree/"
+      + branch;
+  }
+  apiUrl +=
+    '?circle-token=0a5143728784b263d9f0238b8d595522689b3af2'
```

Comment:
It is an API key I created for circle ci, allowing viewing non public repositories. Should this be in secret.json?

Actually, supplying the key is optional for open source projects.